src/org/loklak/api/search/QuoraProfileScraper.java

```diff
@@ -99,6 +101,37 @@ public JSONObject getDefaultPermissions(BaseUserRole baseUserRole) {
         return new HashMap<String, String>();
     }
 
+    protected String prepareSearchUrl(String type) {
+        URIBuilder url = null;
+        String midUrl = "search/";
+        try {
+            switch(type) {
+
+                case "user":
+                    midUrl = "profile/";
+                    url = new URIBuilder(this.baseUrl + midUrl + this.query);
+                    break;
+                case "question":
+                    url = new URIBuilder(this.baseUrl + midUrl);
+                    url.addParameter("q", this.query);
+                    url.addParameter("type", "question");
+                    break;
+                case "answer":
+                    url = new URIBuilder(this.baseUrl + midUrl);
+                    url.addParameter("q", this.query);
+                    url.addParameter("type", "answer");
+                    break;
+                default:
+                    url = new URIBuilder("");
+                    break;
+            }
+        } catch (URISyntaxException e) {
+            DAO.log("Invalid Url: baseUrl = " + this.baseUrl + ", mid-URL = " + midUrl + "query = " + this.query + "type = " + type);
+        }
+
+        return url.toString();
```

Comment:
now if there is any exception the scraper doesn't fails, (though it wasn't failing earlier), but error will be logged with input parameters